plugins/woocommerce-blocks/tests/e2e/tests/price-filter/price-filter.block_theme.side_effects.spec.ts

```diff
@@ -59,24 +59,29 @@ test.describe( `${ blockData.name } Block - with All products Block`, () => {
 		page,
 		frontendUtils,
 	} ) => {
+		// The price filter input is initially enabled, but it becomes disabled
+		// for the time it takes to fetch the data. To avoid setting the filter
+		// value before the input is properly initialized, we wait for the input
+		// to be disabled first. This is a safeguard to avoid flakiness which
+		// should be addressed in the code, but All Products block will be
+		// deprecated in the future, so we are not going to optimize it.
+		await page
+			.getByRole( 'textbox', {
+				name: 'Filter products by maximum price',
+				disabled: true,
+			} )
+			.waitFor( { timeout: 3000 } )
+			// Do not throw in case Playwright doesn't make it in time for the
+			// initial (pre-request) render.
+			.catch();
+
 		const maxPriceInput = page.getByRole( 'textbox', {
 			name: 'Filter products by maximum price',
 		} );
 
-		// All Products block will be deprecated in the future, so we are not going to optimize it.
-
-		// eslint-disable-next-line playwright/no-networkidle
-		await page.waitForLoadState( 'networkidle' );
-
-		await frontendUtils.selectTextInput( maxPriceInput );
-		await maxPriceInput.fill( '$10', {
-			// eslint-disable-next-line playwright/no-force-option
-			force: true,
-		} );
+		await maxPriceInput.dblclick();
+		await maxPriceInput.fill( '$10' );
 		await maxPriceInput.press( 'Tab' );
```

Comment:
This feels weird, not from the test but from the feature implementation perspective. Hitting Tab changes the focus to another element and loses the filter from the sight, so is that how it's intended to work? Why not update on value change or Enter? Not sure who to mention here, actually 😛 cc: @gigitux